protocol/src/main/java/org/eclipse/ditto/protocol/mappingstrategies/AbstractMessageMappingStrategies.java

```diff
@@ -131,12 +129,19 @@ protected static MessageHeaders messageHeadersFrom(final Adaptable adaptable) {
     }
 
     private static void validatePathForLiveMessages(final MessagePath path, final String messageSubject) {
-        final String pathValidationRegex = "(/features/[^/]+)?/(inbox|outbox)/messages/" + messageSubject;
-        final Pattern pattern = Pattern.compile(pathValidationRegex);
-        final Matcher matcher = pattern.matcher(path.toString());
-        if (!matcher.matches()) {
+        final String pathAsString = path.toString();
+        final boolean valid;
+        if (pathAsString.startsWith("/features")) {
+            valid = pathAsString.endsWith("/inbox/messages/" + messageSubject) ||
+                    pathAsString.endsWith("/outbox/messages/" + messageSubject);
+        } else {
+            valid = pathAsString.equals("/inbox/messages/" + messageSubject) ||
+                    pathAsString.endsWith("/outbox/messages/" + messageSubject);
```

Comment:
Shouldn't this be `pathAsString.equals("/outbox/messages/" + messageSubject)` ?

Reply:
Fixed.